crates/analytics/src/lib.rs

```diff
@@ -64,6 +64,14 @@ use self::{
     types::MetricsError,
 };
 
+#[derive(Clone, Debug)]
+pub struct TenantID(String);
```

Comment:
Can you move this struct definition to the common_utils crate?

Reply:
I moved it to types.rs in common_utils crate and added #[allow(unused)] in some places so clippy can stop complaining. Is that fine?